Apps/Sandcastle/gallery/Clamp to 3D Tiles.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
```

Comment:
This is missing the thumbnail?  Perhaps `gallery/development/Clamp to Tileset.jpg` is in the wrong location?